### ISSUE TITLE
SVG Support for imageView

### ISSUE THREAD

**Issue by darkwood-studios** [TheJuki/KFormMaster#152]
Hi there,

I do have a svg file which I want to place into the imageView.

This is my example:

```
var svgImage = "data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxNiAxNiIgd2lkdGg9IjEyOCIgaGVpZ2h0PSIxMjgiPjxwYXRoIGZpbGw9IiM3YjM3MjUiIGQ9Ik0wIDBoMTZ2MTZIMHoiLz48dGV4dCB4PSI4IiB5PSI4IiBmaWxsPSIjZmZmIiB0ZXh0LWFuY2hvcj0ibWlkZGxlIiBkeT0iLjNlbSIgZm9udC1mYW1pbHk9IkFyaWFsIiBmb250LXNpemU9IjciPkRBPC90ZXh0Pjwvc3ZnPg=="

formBuilder = form(ctx, settings_rv) {
            imageView {
                value = svgImage
            }
}
```

The code is not working, because it expects a web url... do you have any idea to get this to work?

Maybe something like this could help https://stackoverflow.com/questions/29375482/how-to-load-an-image-from-dataurl-in-android-imageview - maybe check in FormImageViewBinder if we got a data-url and than convert it to a bitmap.

Best,
Daniel

**Comment by TheJuki** [comment]
I just released [6.4.5](https://github.com/TheJuki/KFormMaster/releases/latest). Please update your dependency and let me know if SVG images from a data: URL works for you. One limitation is that the imageTransformation property does not work for SVGs but will work for other images types.

**Comment by darkwood-studios** [comment]
Great one. Fits well. Thank you very much. :-)
